ml4h/applications/ingest/ingest_mri.py

```diff
@@ -0,0 +1,196 @@
+import pydicom as dicom
+from dicom_parser import Image
+from dicom_parser.utils.siemens.csa.header import CsaHeader
+import numpy as np
+import pandas as pd
+import glob
+import fastparquet as fp
+import blosc
+import xxhash
```

Comment:
Should we add these to our requirements?

Reply:
I don't believe that is appropriate in this case. The core functionality of ML4H has no dependency of these. These are only required in the special case when a user require the ingestion of MRI data from the UKBioBank.